loadtest/beats-yaml-generator.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+stream = os.popen('kubectl get pods -o wide -n logstash')
```

Comment:
These variables are a text book example of bad variable names.  :). Please rename `stream`, `output`, `result`, and `o` to something more descriptive.